### PR TITLE
Tines Rule - Job Deletion (2 cases)

### PR DESCRIPTION
**Pull request by josh-panther**
### Background

https://www.tines.com/api/audit-logs#:~:text=JobsQueuedDeletion,retrying%20jobs%20were%20cleared%20(for%20a%20self%2Dhosted%20tenant)

### Changes

Tines - JobsQueuedDeletion, JobsRetryingDeletion

### Testing

```
Tines.Enqueued.Retrying.Job.Destruction
	[PASS] Trigger - JobsQueuedDeletion
		[PASS] [rule] true
		[PASS] [title] Tines [JobsQueuedDeletion] performed by [user@email.com] on [tines-log-source-name].
		[PASS] [dedup] Tines [JobsQueuedDeletion] performed by [user@email.com] on [tines-log-source-name].
		[PASS] [alertContext] {"actor": "user@email.com", "action": "JobsQueuedDeletion", "tenant_id": "1337", "user_email": "user@email.com", "user_id": "7331", "operation_name": "JobsQueuedDeletion", "request_ip": "98.224.225.84"}
	[PASS] Trigger - JobsRetryingDeletion
		[PASS] [rule] true
		[PASS] [title] Tines [JobsRetryingDeletion] performed by [user@email.com] on [tines-log-source-name].
		[PASS] [dedup] Tines [JobsRetryingDeletion] performed by [user@email.com] on [tines-log-source-name].
		[PASS] [alertContext] {"actor": "user@email.com", "action": "JobsRetryingDeletion", "tenant_id": "1337", "user_email": "user@email.com", "user_id": "7331", "operation_name": "JobsRetryingDeletion", "request_ip": "98.224.225.84"}
	[PASS] Tines Login
		[PASS] [rule] false
```

### CODE REVIEW

File: rules/tines_rules/tines.enqueued_retrying.job.deletion.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def rule(event):
+    return event.get('operation_name') in ["JobsQueuedDeletion", "JobsRetryingDeletion"]
+
+def title(event):
+    operation = event.get('operation_name')
+    user = event.get('user_email')
+    tines_instance = event.get('p_source_label')
```
Comment (edyesed) [inline rules/tines_rules/tines.enqueued_retrying.job.deletion.py:7]: please use `deep_get()` with a `default=` kwarg  in place of `event.get()`

File: rules/tines_rules/tines.enqueued_retrying.job.deletion.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    return f"Tines {operation} by {user} on {tines_instance}"
```
Comment (edyesed) [inline rules/tines_rules/tines.enqueued_retrying.job.deletion.py:8]:
```suggestion
    return f"Tines [{operation}] by [{user}] on [{tines_instance}]"
```

**Review by edyesed — CHANGES_REQUESTED** [review]
Please add an `ExpectedResult: false` test case and set the alert_context using the [tines global helper alert_context](https://github.com/panther-labs/panther-analysis/blob/master/global_helpers/panther_tines_helpers.py#L1)

**Comment by josh-panther** [comment]
Waiting for https://github.com/panther-labs/panther-analysis/pull/786 before review req

**Review by edyesed — APPROVED** [review]
adds detection for tines operations that relate to data destruction

**Pull request merged**